examples/property_estimator/start_client.py

```diff
@@ -8,22 +8,43 @@
 from propertyestimator.utils import get_data_filename, setup_timestamp_logging
 
 
-def submit_calculation_to_server():
+def request_estimate_blocking():
     """Submit calculations to a running server instance"""
+    setup_timestamp_logging()
+
+    # Load in the data set of interest.
+    data_set = ThermoMLDataSet.from_file_list(get_data_filename('properties/single_density.xml'))
+    # Load in the force field to use.
+    force_field = smirnoff.ForceField(get_data_filename('forcefield/smirnoff99Frosst.offxml'))
+
+    # Create the client object.
+    property_estimator = client.PropertyEstimatorClient()
+    # Submit the request to a running server, and wait for the results.
+    result = property_estimator.estimate(data_set, force_field)
```

Comment:
Is this the blocking- or non-blocking call? `estimate()` is good example of a clear, direct method name for a blocking call.